tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -604,6 +604,21 @@ impl<C, T, W> Subscriber<C, format::JsonFields, format::Format<format::Json, T>,
             ..self
         }
     }
+
+    /// Sets whether or not the formatter will include newline characters after
+    /// formatting a log event.
+    ///
+    /// See [`format::Json`]
```

Comment:
maybe this should link directly to the section on wrapping the JSON formatter you added? also, we could include the language from the `Json::with_newlines` method here as well:
```suggestion
    ///
    /// This option is mainly useful for logic that is supposed to expand logged
    /// JSON values by embedding them in a wrapping JSON structure. See 
    /// [the `format::Json` documentation][wrapping] for details on this.
    ///
    /// [wrapping]: format::Json#wrapping-json-entries-with-custom-formatters
```